README.md

```diff
@@ -58,20 +58,22 @@ Ensure you have the necessary dependencies:
 - `crystal`: Follow the instructions to get `crystal` on this page: <https://crystal-lang.org/docs/installation/index.html>
 
 ##### For Debian & Ubuntu
-- These are necessary to compile the CLI:
-- `sudo apt-get install build-essential libreadline-dev libsqlite3-dev libpq-dev libmysqlclient-dev libssl-dev`
+
+These are necessary to compile the CLI:
+
+- apt-get install build-essential libreadline-dev libsqlite3-dev libpq-dev libmysqlclient-dev libssl-dev`
 
 <!-- WIP: ##### For RedHat & CentOS
 - `sudo yum groupinstall 'Development Tools' `
 - `sudo yum install readline-devel sqlite-devel openssl-devel libyaml-devel gc-devel libevent-devel` -->
 
-Once you have these dependencies, You can build the `amber` tool from source:
+Once you have these dependencies, you can build `amber` from source:
 
 
 ```shellsession
 $ git clone git@github.com:amberframework/amber.git
-$ cd amber/
-$ shards install
+$ cd amber
+$ make
```

Comment:
you have to run the shards install before running make

Reply:
It seems that this is related to this change https://github.com/amberframework/amber/pull/484/files#diff-b67911656ef5d18c4ae36cb6741b7965R14

If so running `make install` is sufficient here?

Reply:
Yes, 'make install' would download deps, compile, and install. (Assuming that the privileges under which process is running are enough to do an installation in /usr/local/bin)